cluster-provision/k8s/1.17/ovsdpdk.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -ex
+
+setenforce 0
+
+# TODO: Can this be moved to privision script?
+# Having these packages installed does not have impact on the regular deployments
+yum install -y dpdk dpdk-devel
+
+# Container image is creted using the build scripts in repo https://github.com/krsacme/container-builds-scripts
+# By using the prebuilt driver and rpm, the time is reduced in CI to build it
+KUBEVIRT_OVSDPDK_BUILD=${KUBEVIRT_OVSDPDK_BUILD:-"false"}
+KUBEVIRT_OVSDPDK_HELPER_IMAGE=${KUBEVIRT_OVSDPDK_HELPER_IMAGE:-"quay.io/krsacme/kubevirt-ovsdpdk-helpers:latest"}
```

Comment:
Are you considering to move the image into the kubevirtci docker repo at a later point in time?

Reply:
> I understand that you are planning to install dpdk libs on the cluster nodes everytime the cluster is spun up, right?
> 
> Wouldn't it make sense to create a separate provider (i.e `k8s-1.17-dpdk`) so to reduce spin up time instead of having to install everything everytime you do `cluster-up`?

Thanks @dhiller for the comments. 

Yes, it would be ideal if we are able to create ``k8s-1.x-dpdk`` images for each release. I am unable to idenify how these images are built (actual scripts to run the ``provision`` command to create the ``k8s.1.17`` image). Can you point how these images are created and pushed?

One drawback for a separate image is, the release process will have to depend on the DPDK version and its compatibility with the CentOS version. Though it is not a riskier option (because we use only LTS version), I am not sure if it is agreeable to include this dependency in the general release process of a new version. If it is acceptable to create a new image, then the same script can be invoked by ``provision`` command instead of ``run`` command in the follow-up patch.

Yes, if we invoke this as part of the ``run`` without creating a new image, it will include additional CI run time, for DPDK jobs. Here are the major steps:
1) Update kernel package
2) Reboot
3) Pull Docker image for kernel module and ovs rpm
4) OvS RPM install

All these steps increse the time around 4-5 mins based on the CentOS repo sync.

> Are you considering to move the image into the kubevirtci docker repo at a later point in time?

If the seprate image creating is acceptable, then we can remove this container and build the image DPDK everytime the ``k8s.1.17-dpdk`` image is created. In case it is not agreeable solution, then yes, I will plan to move this container image and the repo to build this container image [container-build-scripts](https://github.com/krsacme/container-builds-scripts) in to kubevirt umberla.